tests/utils/test_parallel_embeddings.py

```diff
@@ -474,3 +476,67 @@ def test_chimera_on_pegasus(self):
         m_nice = n_nice = m - 1  # A nice subgraph is relevant to Chimera sublattices:
         expected_number_of_cells = (m_nice // m_sub) * (n_nice // 3) * 3
         self.assertEqual(len(embeddings), expected_number_of_cells)
+
+    def test_T_family_T_kwargs(self):
+        # Like test basic, but pass family information
+        with_Ts = [True, False]
+        with_Tfamily = [True, False]
+        for topology, with_T, with_Tfamily in product(
+            ["chimera", "pegasus", "zephyr"], with_Ts, with_Tfamily
+        ):
```

Comment:
To use `parameterized`, add `from parameterized import parameterized` at the top, and then:
```suggestion
    @parameterized.expand(
        product(["chimera", "pegasus", "zephyr"], [False, True], [False, True])
    )
    def test_T_family_T_kwargs(self, topology, with_T, with_Tfamily):
        # Like test basic, but pass family information
```
Also, need to dedent the ex for body.